app/views/red_line_boundary_change_validation_requests/show.html.erb

```diff
@@ -21,14 +21,14 @@
     <p class="govuk-body">
       <%= @red_line_boundary_change_validation_request.reason  %>
     </p>
-    <%= render "shared/location_map", locals: { geojson: @planning_application.boundary_geojson } %>
+    <%= render "shared/location_map", locals: { geojson: @red_line_boundary_change_validation_request.original_geojson } %>
 
     <h2 class="govuk-heading-m govuk-!-padding-top-6">
      Proposed red line boundary
     </h2>
     <%= render "shared/location_map", locals: { geojson: @red_line_boundary_change_validation_request.new_geojson } %>
```

Comment:
we need to prevent some errors on this bit, some ideas:
1. Update the db for old red_line boundaries to have both old and new geojson (the old geojson can be set to the current one for ex (like this migration update ex: db/migrate/20210910145259_add_reply_to_local_authority.rb)

2. Add a little if statement to make sure this doesn't blow up if no "old" and "new" fields exist

3. Wrap this in a helper method that renders a default string for "No previous geojson" available if the view doesn't exist else render partial